### DISCUSSION TITLE
turecolor support

### DISCUSSION THREAD

**Discussion by jxcangel** [kingToolbox/WindTerm#508] [General]
for zsh/bash shell, it support turecolor by:

```
export TERM=xterm-direct
```

and tmux support turecolor also:

```
set-option -g default-terminal "xterm-256color"
set-option -sa terminal-overrides ",xterm*:Tc"
```

based on 2.2.2, ssh session only set terminal mode to xterm-256color, can we support xterm-direct or others?

**Comment by kingToolbox** [comment]
Although the name is `xterm-256color`, it actually already supports `truecolor`, as shown in the [MainWindow screenshot](https://github.com/kingToolbox/WindTerm/raw/master/images/screenshots/WindTerm.png). Usually the server side supports `xterm` and `xterm-256color`, so there is no special need to support `xterm-direct`.

If your server only supports `xterm-direct`, then it is also very simple. You just need to copy `WindTerm_installed_path/terminal/terms/xterm-256color.term` and rename it to `WindTerm_installed_path/terminal/terms/xterm-direct.term`, then restart `WindTerm`, and then you can select `xterm-direct` in the `Term` drop-down box.
